Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:8-jre
+WORKDIR /opt/brvs
+COPY ./dapp-core/build/libs/iroha-dapp-all.jar /opt/dapp/dapp.jar
+
+## THE LIFE SAVER
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.5.0/wait /wait
+RUN chmod +x /wait
+## Launch the wait tool and then the application
+ADD ./dapp_run.sh /dapp_run.sh
+ENTRYPOINT ["/dapp_run.sh"]
```

Comment:
If you'll use 'ENTRYPOINT ["sh","/data-collector_run.sh"]' you will not need to add 'chmod +x'

Reply:
Ok, modified